fabric-item-api-v1/src/main/java/net/fabricmc/fabric/impl/item/FabricItemInternals.java

```diff
@@ -18,13 +18,15 @@
 
 import java.util.WeakHashMap;
 
+import net.fabricmc.fabric.api.registry.FuelRegistry;
 import net.minecraft.item.Item;
 
 import net.fabricmc.fabric.api.item.v1.CustomDamageHandler;
 import net.fabricmc.fabric.api.item.v1.EquipmentSlotProvider;
 
 public final class FabricItemInternals {
 	private static final WeakHashMap<Item.Settings, ExtraData> extraData = new WeakHashMap<>();
+	private static int fuelCookTime = -1;
```

Comment:
This should be stored in an an `ExtraData` instance so that it doesn't get applied to all items created after it. See `FabricItemSettings#equipmentSlot()`

Reply:
Done!